pipeline_steps/common.groovy

```diff
@@ -1625,35 +1625,44 @@ void createComponentGateTrigger(String name, String repoUrl, String jiraProjectK
   }
 }
 
+List getMatchingJobs(regexMatch){
```

Comment:
We pass in a string here, can set the type to `String`.

Reply:
Good call, will do that.